tests/Datagrid/PagerTest.php

```diff
@@ -136,11 +136,14 @@ public function testGetMaxRecordLimit(): void
         $this->assertSame(99, $this->pager->getMaxRecordLimit());
     }
 
+    /**
+     * @group legacy
```

Comment:
Add a `NEXT_MAJOR: Remove this test and the setProperty method`.

Reply:
`setProperty` will still be used to set the private property then on master

Reply:
Not if we're doing this: https://github.com/sonata-project/SonataAdminBundle/pull/6732#discussion_r551461675

Reply:
Ok this whole test needs refactoring then I guess since it needs an implemented `countResults` method but its mocking the abstract Pager